docs/spark/README.md

```diff
@@ -0,0 +1,174 @@
+# Apache Spark Usage within Raster Foundry
+
+Apache Spark is a cluster computing system with a Scala API for defining execution graphs and an engine for executing them across multiple machines. GeoTrellis is a library built on top of Spark for describing geospatial transformations against raster data sets.
+
+Raster Foundry uses both to mosaic multiple large raster data sets and output the result in a way that can be consumed by a web UI.
+
+## Table of Contents
+
+* [Spark Components](#spark-components)
+  * [Master](#master)
+  * [Worker](#workers-and-executors)
+  * [Driver](#driver)
+* [Master Recovery](#master-recovery)
+* [Development Environment](#development-environment)
+  * [Building a Job JAR](#building-a-job-jar)
+  * [Local Spark Standalone Cluster](#local-spark-standalone-cluster)
+* [Safety Testing](#safety-testing)
+
+## Spark Components
+
+Often times Spark is deployed on top of a cluster resource manager, such as Apache Mesos or Apache Hadoop YARN (Yet Another Resource Negotiator). In addition to those options, Spark comes bundled with **Spark Standalone**, which is a built-in way to run Spark in a clustered environment.
```

Comment:
Is the idea that we are going to deploy docker containers and rely on the standalone cluster manager to manage it? Is the reason for this reduce complexity of not having a cluster manager such as Mesos or YARN, or are there other benefits?